sdk/core/azure-core/src/main/java/com/azure/core/util/Configuration.java

```diff
@@ -158,11 +162,38 @@ public class Configuration implements Cloneable {
      */
     public static final String PROPERTY_AZURE_TRACING_DISABLED = "AZURE_TRACING_DISABLED";
 
+    /**
+     * Sets the name of the {@link TracerProvider} implementation that should be used to construct instances of
+     * {@link Tracer}.
+     * <p>
+     * The name must be the full class name, e.g. {@code com.azure.core.tracing.opentelemetry.OpenTelemetryTracerProvider} and not
+     * {@code OpenTelemetryTracerProvider}.
+     * <p>
+     * If the value isn't set or is an empty string the first {@link TracerProvider} resolved by {@link java.util.ServiceLoader} will be
+     * used to create an instance of {@link Tracer}. If the value is set and doesn't match any
+     * {@link TracerProvider} resolved by {@link java.util.ServiceLoader} an {@link IllegalStateException} will be thrown when
+     * attempting to create an instance of {@link TracerProvider}.
+     */
+    public static final String PROPERTY_AZURE_TRACER_PROVIDER_IMPLEMENTATION = "AZURE_TRACER_PROVIDER_IMPLEMENTATION";
```

Comment:
Should we use `NAME` instead of `IMPLEMENTATION` to make it clear that it's just the name?

Reply:
I was following the pattern of `PROPERTY_AZURE_HTTP_CLIENT_IMPLEMENTATION` we already have and now made it closer with `PROPERTY_AZURE_TRACING_IMPLEMENTATION`. Do you feel it's good enough considering we already have one like that for HTTP clients?